I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/generation.rpy", line 209, in script call
    python:
  File "game/script.rpy", line 26, in script call
    $ _allChars = allChars
  File "game/script.rpy", line 26, in script
    $ _allChars = allChars
  File "game/script.rpy", line 26, in <module>
    $ _allChars = allChars
NameError: name 'allChars' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/generation.rpy", line 209, in script call
    python:
  File "game/script.rpy", line 26, in script call
    $ _allChars = allChars
  File "game/script.rpy", line 26, in script
    $ _allChars = allChars
  File "E:\RenPy\renpy-6.99.1-sdk\renpy\ast.py", line 785, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "E:\RenPy\renpy-6.99.1-sdk\renpy\python.py", line 1432, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 26, in <module>
    $ _allChars = allChars
NameError: name 'allChars' is not defined

Windows-7-6.1.7601-SP1
Ren'Py 6.99.1.329
HHS 0.1
